### PR TITLE
Portfolio Gallery Start

### PR DESCRIPTION
**Pull request by Cindyzhang977**
Create initial layout/design of gallery portion of portfolio

- grid layout for photos
- 3 photos to start 
- location & date to caption each photo 
- add front end of "sort by" and "filter" dropdowns

### CODE REVIEW

**Comment by Cindyzhang977** [comment]
image

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -3,16 +3,90 @@
...
+                    <a href="https://cindyzhang977.github.io/cindyxzhang/">More about me</a>
+                </div>
+            </div>
+        </div>
+
+        <div class="container" id="gallery">
+            <div class="container">
+                <div class="row">
+                    <div class="dropdown">
+                        <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```
Comment (mecolmg) [inline portfolio/src/main/webapp/index.html:30]: While optional consider the line length of your HTML and formatting it via one of the recommended approaches:
https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,86 @@
...
+/* header styles */
+
+#header {
+    max-width: 800px;
+}
+
+#profile-img {
+    border-radius: 50%;
+    height: 200px;
+    object-fit: cover;
```
Comment (mecolmg) [inline portfolio/src/main/webapp/style.css:40]: nit: `object-fit` [isn't supported in IE](https://caniuse.com/#feat=object-fit). A typical workaround for this is to add a wrapper div for the image, and apply the `border-radius: 50%;` on that along with an `overflow: hidden;` attribute.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,86 @@
...
+
+#title {
+    color: var(--navy);
+    font-family: 'Dancing Script', cursive, sans-serif;
+}
+
+/* gallery styles */
+
+.dropdown > .btn {
+    background-color: whitesmoke;
```
Comment (mecolmg) [inline portfolio/src/main/webapp/style.css:52]: `whitesmoke` is an awesome name for a color.

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -1,9 +1,86 @@
...
+    color: black;
+    border-color: gray;
```
Comment (mecolmg) [inline portfolio/src/main/webapp/style.css:54]: Please sort attributes alphabetically.

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -1,9 +1,86 @@
+:root {
+    --navy: #1b2d45;
```
Comment (mecolmg) [inline portfolio/src/main/webapp/style.css:2]: Please update your editor to use 2 spaces for indentation.

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -10,78 +10,84 @@
...
+                <a class="dropdown-item" href="#">Beach</a>
+                <a class="dropdown-item" href="#">Sunset</a>
+                <a class="dropdown-item" href="#">Mountain</a>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="row justify-content-center">
+
+            <!-- Deleted in a future PR -->
```
Comment (mecolmg) [inline portfolio/src/main/webapp/index.html:56]: You can delete it in this PR and rebase the future one to have the changes.

**Pull request merged**